### ISSUE TITLE
Adding WebHook for status changes

### ISSUE THREAD

**Issue by mizrachiran** [arendst/Sonoff-Tasmota#2220]
Hi,

This is a feature request for adding a configurable WebHook http endpoint, to be invoked with the current status payload whenever something change.

Thanks

**Comment by Jason2866** [comment]
Why?

**Comment by mizrachiran** [comment]
When you control sonoff devices through http calls you don’t have the option to know if something change. 
The only alternative is polling which not as good.

**Comment by Jason2866** [comment]
Ok i understand. The basic idea and concept from Tasmota is to control the devices via mqtt...
So if you want a solution via http Tasmota is the wrong choice.
I dont believe @arendst will invest much time and effort for doing it via http

**Comment by mizrachiran** [comment]
Why?
It’s a great feature to have. 
I really think you should consider it.

**Comment by mizrachiran** [comment]
There are lot of home automation hubs which doesn’t support mqtt. And some does not support polling either. 

Adding such feature would make Tasmota more powerful.

**Comment by Jason2866** [comment]
No, it is not a great feature. Do you really read the arguments from @arendst in  #2222  ?

**Comment by mizrachiran** [comment]
Yes, even before i opened this one. 
I still think it’s a great feature to have as it will enable many other scenarios for using Tasmota

**Comment by mizrachiran** [comment]
@luisfpinto

**Comment by luisfpinto** [comment]
Many current smart home and tech solution use Webhooks HTTP endpoint to send a message when a device status changed.  IFTTT, for example, uses this as a trigger for many devices. 

However, I can understand that Tasmota approach to this is more MQTT based but the solution pointed by @mizrachiran makes complete sense.

**Comment by lobradov** [comment]
Why not cont

//_ on phone

**Comment by lobradov** [comment]
(hate when this happens).

Why not contribute it yourself and maybe Theo will include it as optional
code?
if he already doesn't like the idea, he'll hardly do it on his own, but if
code is there...

**Comment by FrankBolleri** [comment]
Hello All,
just a comment to help people searching for this matter and landing on this page in the future.
Seeing at #6692, invoke of an URL using rules currently is possible.
